operations/app/terraform/modules/data_factory/datasets.tf

```diff
@@ -5,7 +5,7 @@ resource "azurerm_data_factory_dataset_binary" "pdi_datasa" {
   linked_service_name = azurerm_data_factory_linked_service_azure_blob_storage.pdi_datasa.name
 
   azure_blob_storage_location {
-    container                = "bronze"
+    container                = "bronze/additional-records"
```

Comment:
It's definitely possible I'm reading this wrong, but I believe `SFTPBinarySink` should still point to `bronze` while `SFTPBinarySink_trial` (which has now been renamed to `Azure_Data_Lake` to be more descriptive) should point to `bronze-additional-records`. Is that right? `bronze/additional-records` is not a container/directory that should exist.

Reply:
Also, starting on line 25, there are resources that are defined that I've since deleted from the Data Factory. `DestinationDataset_mzn` no longer exists. I know it's not part of this PR, but just wanted to point it out in case you wanted to modify that as well.